ts/Series/PackedBubble/PackedBubbleSeries.ts

```diff
@@ -1151,6 +1167,26 @@ class PackedBubbleSeries extends BubbleSeries implements Highcharts.DragNodesSer
         return arr;
     }
 
+    public pointAttribs(
+        point?: PackedBubblePoint,
+        state?: StatesOptionsKey
+    ): SVGAttributes {
+        let options = this.options,
+            hasParentMarker = point && point.isParentNode,
+            markerOptions = hasParentMarker && options.layoutAlgorithm && options.layoutAlgorithm.parentNodeOptions ?
```

Comment:
In general, I would rather go with ternary operator with simple boolean checks, like `oneVariable` ? `optionA` : `optionB`. If we need to check 3 variables in ternary operator, then good idea might be to prepare a separate boolean checking all of it before ternary operator (or simply using an `if` statement)